public/js/main.js

```diff
@@ -1,217 +1,227 @@
 (function($) {
     "use strict";
 
-/*-- Menu Sticky --*/
-var $window = $(window);
-$window.on('scroll', function() {    
-	var scroll = $window.scrollTop();
-	if (scroll < 300) {
-		$(".sticker").removeClass("stick");
-	}else{
-		$(".sticker").addClass("stick");
-	}
-});
-/*-- Mobile Menu --*/
-$('.main-menu').meanmenu({
-	meanScreenWidth: '991',
-	meanMenuContainer: '.mobile-menu',
-	meanMenuClose: '<i class="pe-7s-close-circle"></i>',
-	meanMenuOpen: '<i class="pe-7s-menu"></i>',
-	meanRevealPosition: 'right',
-	meanMenuCloseSize: '30px',
-});
+    /*-- Menu Sticky --*/
+    var $window = $(window);
+    $window.on('scroll', function() {
+        var scroll = $window.scrollTop();
+        if (scroll < 300) {
+            $(".sticker").removeClass("stick");
+        }else{
+            $(".sticker").addClass("stick");
+        }
+    });
+    /*-- Mobile Menu --*/
```

Comment:
meaning here

Reply:
Ahhh! no. That statement was already there from the template.
I did ctrl+A -> ctrl-X -> ctrl-V on the main.js file so it aligned the whole file, adding spaces before some statements that's why. Na phpstorm nor boy.